stackdriver_exporter.go

```diff
@@ -57,6 +57,10 @@ var (
 		"google.project-id", "Comma seperated list of Google Project IDs.",
 	).String()
 
+	projectsQuery = kingpin.Flag(
+		"google.projects.filter", "Google projects search query.",
```

Comment:
Minor nit, let's name this consistently with the description.

```suggestion
		"google.projects-query", "Google projects search query.",
```

Reply:
I'm gonna change all to `filter` because that's the Gcloud command line option name.